browser_patches/firefox/juggler/content/PageAgent.js

```diff
@@ -801,6 +790,8 @@ class PageAgent {
     const element = window.windowUtils.elementFromPoint(x, y, false, false);
     const event = window.document.createEvent('DragEvent');
 
+    const dataTransfer = dragService.getCurrentSession().dataTransfer.mozCloneForEvent(type);
+    dataTransfer.dropEffect = 'move';
```

Comment:
is this necessary?

Reply:
tests didn't pass without it! 🤷‍♂️ I'll ask Joel when he's back